static/language/en/showcase/lexImpact.json

```diff
@@ -0,0 +1,20 @@
+LexImpact
+
+Back
+
+Simulate the impact of tax reforms on citizens. 
+
+Powered by Etalab and part of the EIG program (General Interest Entrepreneur), the team works with admninistrator within the French National Assembly.
+
+This project aims at helping administrators and deputees to simulate the impact of reforms on the government budget before its vote. 
```

Comment:
```suggestion
This project aims at helping administrators and deputees of the French parliements to simulate the impact of socio-fiscal amendment before its vote. 
```

Reply:
Assembly already mentioned on the previous sentence